grpc-api-client/src/main/java/io/deephaven/grpc_api_client/table/BarrageSourcedTable.java

```diff
@@ -436,6 +441,10 @@ private synchronized void realRefresh() {
         }
 
         if (sealed) {
+            if (onSealRunnable != null) {
+                onSealRunnable.run();
```

Comment:
probably stupid question, but what if we seal but never call this, is that possible? seal racing with release for example?

and what if the runnable throws? (for example, the export func runs synchronously, and onCompleted throws)